graph/loaders/assets.js

```diff
@@ -65,6 +76,7 @@ module.exports = (context) => ({
     // this operation create a new asset if one isn't found.
     getByURL: (url) => findOrCreateAssetByURL(context, url),
 
+    search: (value, limit) => getAssetsByQuery(context, value, limit),
```

Comment:
Same as above, instead of putting the query variables into separate arguments, just pass a query object.